tests/dbus/_misc.py

```diff
@@ -75,10 +78,127 @@ def __init__(self, not_sim=False):
 
     def setUp(self):
         if MODE == SIMULATOR and not self._not_sim:
-            self._stratisd = subprocess.Popen([os.path.join(_STRATISD), '--sim'])
+            self._stratisd = subprocess.Popen([os.path.join(_STRATISD),
+                                               '--sim'])
         else:
             self._stratisd = subprocess.Popen([os.path.join(_STRATISD),
                                                '--debug'])
 
 
 Service = ServiceR
+
+
+BIN = os.getenv('STRATIS_LOSETUP_BIN', "/usr/sbin/losetup")
```

Comment:
You could put this all in a separate file, to real benefit. For now I"m ignoring it, I"m assuming it works correctly.